modules/core/src/experimental/utils/gpu-grid-aggregation/grid-aggregation-utils.js

```diff
@@ -15,6 +15,7 @@ export function pointToDensityGridData({
   gpuGridAggregator,
   gpuAggregation,
   aggregationFlags,
+  getWeight = null,
```

Comment:
Can we use a defined weight over `null` here? like the default accessors in the layers.
e.g.:
```
getWeight = d => [1.0, 0, 0],
```